app/experimenter/visualization/api/v3/views.py

```diff
@@ -25,7 +25,7 @@ def load_data_from_gcs(path):
     return (
         json.loads(default_storage.open(path).read())
         if default_storage.exists(path)
-        else []
+        else None
     )
```

Comment:
Oh I just realized you could also write this as

```py
if default_storage.exists(path):
    return json.loads(default_storage.open(path).read())
```

since a function will return `None` by default